dev-python/qtawesome/qtawesome-0.6.1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+MY_PN="QtAwesome"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Enables iconic fonts such as Font Awesome in PyQt"
+HOMEPAGE="https://github.com/spyder-ide/qtawesome/ https://pypi.org/project/QtAwesome/"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="test"
+
+RDEPEND="media-fonts/fontawesome
+		dev-python/QtPy[${PYTHON_USEDEP}]
+		dev-python/six[${PYTHON_USEDEP}]"
```

Comment:
Extra tabs.

Reply:
Should it be just one tab? or does it need an extra third tab?

Personally I think 2 tabs looks best (at least it does in my editor)

Reply:
Yeah, one tab there.
https://github.com/gentoo/policy-guide/blob/master/ebuild-format.rst#coding-style

Reply:
Alright, I've fixed that here, and in my other sub-PRs. I'll fix this in the spyder master-PR next time I push to that.